src/logic/player.h

```diff
@@ -437,7 +437,7 @@ class Player {
 	};
 
 	const Field* fields() const {
-		return fields_;
+		return fields_.get();
```

Comment:
Please comment what these fields are like:
std::unique_ptr<Field[]> fields_; // Fields currently visisble to the player or (aprtial) last known Status of a Fields

Does unique_ptr not have a * operator that maps to .get() anyway?
See: http://www.cplusplus.com/reference/memory/unique_ptr/operator*/

Reply:
Previously this was a raw `Field*`
Now I'm using `unique_ptr<Field*>`
The `*` operator returns the underlying `Field&` (equal to `[0]`) while here i need `get()` to return the `Field*` (array)